### PR TITLE
Fixed compatibility with Symfony3

### PR DESCRIPTION
**Pull request by kejwmen**

### CODE REVIEW

File: Form/Extension/DisableCsrfExtension.php [hunk 1]
```diff
@@ -95,6 +96,6 @@ public function configureOptions(OptionsResolver $resolver)
      */
     public function getExtendedType()
     {
-        return 'form';
+        return FormType::class;
```
Comment (dunglas) [inline Form/Extension/DisableCsrfExtension.php]: It doesn't work with PHP >= 5.5 and we still support PHP 5.3. Can you use the class name as a string instead please?

**Comment by dunglas** [comment]
Can you also squash your commits please?

**Comment by kejwmen** [comment]
done

**Comment by dunglas** [comment]
Thank you very much @kejwmen

**Pull request merged**